scripts/build/gen-containerd-config.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+sudo curl -LJO https://github.com/containerd/containerd/releases/download/v1.7.0/containerd-1.7.0-linux-amd64.tar.gz
+sudo mkdir bin && sudo tar -C bin -xzvf containerd-1.7.0-linux-amd64.tar.gz
+
+sudo tee bin/config.toml > /dev/null <<EOF
```

Comment:
Is it necessary to add `sudo` in front of every commands? I mean it is possiable to add it in `job.step.run`, https://github.com/kuasar-io/kuasar/pull/95#discussion_r1337032458